lib/Builtins.cpp

```diff
@@ -14,171 +14,335 @@
 
 #include "Builtins.h"
 
+#include <stdlib.h>
```

Comment:
#include <cstdlib> please

Reply:
I don't follow?

Reply:
Use \<cstdlib\> instead of <stdlib.h> preferably. I think the markdown formatting lost cstdlib in my last comment, sorry.